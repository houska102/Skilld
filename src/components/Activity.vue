<template lang="html">
  <div class="activity">
    <span>{{activity.date}}</span><span>{{activity.time}}h</span>
  </div>
</template>

<script>
export default {
  props: {
    activity: {
      type: Object
    }
  }
}
</script>

<style lang="css" scoped>
.activity{
  padding:5px;
  display:flex;
  justify-content: space-between;
}
</style>
