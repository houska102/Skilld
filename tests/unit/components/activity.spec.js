import { shallowMount, createLocalVue } from '@vue/test-utils'
import Activity from "@/components/Activity"

const localVue = createLocalVue()

describe('Skill component test suite', () => {
  let activity
  beforeEach(() => {
    activity = shallowMount(Activity, {
      localVue,
      propsData: {
        activity: { id: 1, date: '26-1-2019', time: 0.5 }
      }
    })
  })

  test('activity component compiles', () => {
    expect(activity).toBeDefined()
  })

  test('activity component compiles', () => {
    expect(activity.element).toMatchSnapshot()
  })

})
