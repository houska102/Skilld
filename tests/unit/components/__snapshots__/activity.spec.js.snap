// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skill component test suite activity component compiles 1`] = `
<div
  class="activity"
>
  <span>
    26-1-2019
  </span>
  <span>
    0.5h
  </span>
</div>
`;
